don't duplicate normative requirements from updates Bundle profile (#612)
HL7/fhircast-docs: input/pagecontent/3-6-3-DiagnosticReport-update.md

#### Supported Update Request Methods

> Each `entry` in the `updates` Bundle resource contains one of the below `method` values in an entry's `request` attribute. The [Bundle profile](StructureDefinition-fhircast-content-update-bundle.html) specifies the allowed methods: PUT can be used to add a new resource or update an existing resource, DELETE removes an existing resource. 

Author: ```suggestion
Each `entry` in the `updates` Bundle resource contains a `method` in that entry's `request` attribute. The [Bundle profile](StructureDefinition-fhircast-content-update-bundle.html) specifies the allowed methods: PUT can be used to add a new resource or update an existing resource, DELETE removes an existing resource. 

```
